src/dune_engine/vcs.ml

```diff
@@ -62,7 +62,8 @@ let prog t =
 let run t args =
   let open Fiber.O in
   let+ s =
-    Process.run_capture Strict (prog t) args ~dir:t.root ~env:Env.initial
+    Process.run_capture ~display:Display.quiet Strict (prog t) args ~dir:t.root
```

Comment:
@snowleopard This one I believe.

Reply:
Aha! Surely this change can go into a separate PR for the actual bug fix?

Reply:
(Unless I'm mistaken and it's not actually easy to leave the old/broken behaviour unchanged in the refactoring PR?)

Reply:
I suppose we could factor it so that they all point to the global clflag option, and then have another PR making them quiet. It's just more work for me (-: